common/faq.html

```diff
@@ -257,4 +257,18 @@ <h2 class="usa-accordion__heading">
     <div id="c6" class="usa-accordion__content">
       Good news: our sandbox environment matches the production environment, and we'll roll out new functionality in sandbox before sending it into production. Even if you are not able to participate in the production pilot at this juncture, please continue to test in the sandbox environment with synthetic data. Your feedback through the Google Group is invaluable to our researchers and engineers as we all work together to shape the future of DPC.
     </div>
+
+    <h2 class="usa-accordion__heading">
+      <button class="usa-accordion__button" aria-expanded="false" aria-controls="c7">
+        Will AWS/SF HITRUST certification satisfy the Security requirements for participation in production DPC?
+      </button>
+    </h2>
+    <div id="c7" class="usa-accordion__content">
+      Yes, DPC implementers can inherit AWS’ HITRUST certification if their DPC solution meets the following requirements:
+      <ol>
+        <li>Their DPC solution is contained entirely within AWS Services that have received a HITRUST certification. See AWS' list of <a href="https://aws.amazon.com/compliance/services-in-scope/">HITRUST-certifided services</a> under HITTRUST CSF.</li>
```

Comment:
typo in link - should be certified not certifided. HITTtust at the end also has an extra T